src/assets/stylesheets/index.scss

```diff
@@ -404,19 +404,6 @@ h6 {
   margin-bottom: 0.7rem;
 }
 
-div {
```

Comment:
* [ ] The styling of "Label headline" and "Label subtitle" changes when switching to the no-heading level. Changing the SEO level should never change the chosen style.
<img width="100%" alt="image" src="https://github.com/user-attachments/assets/7f3335a1-89e6-4745-bdad-0109f83c4a10">
<img width="100%" alt="image" src="https://github.com/user-attachments/assets/8dacc927-9bbc-4ebd-ab74-66541a7177f1">

Reply:
I think this is an independent issues with `Label heading` and `Label subtitle` that already existed prior to this PR. I think it should be addressed separately, once our designer is available again.

Reply:
I'm confused. We introduce `div` headlines in this PR. How is it even possible for a `div` headline related issue to have existed before that?

Reply:
`Label heading` and `Label subtitle` will already (prior to this PR) automatically select `<div>` if no heading tag is set.

Reply:
I see, it's the "unset" case. Thanks for clarifying!